testing/hiawatha/hiawatha.initd

```diff
@@ -2,12 +2,21 @@
 
 supervisor=supervise-daemon
 
-name="Webserver"
-description="Secure and Advanced Hiawatha Webserver"
+name="Hiawatha webserver"
+description="Secure and advanced webserver"
 
 command=/usr/sbin/hiawatha
-command_args_foreground="-d"
+command_args_foreground="-d 1>/dev/null"
+pidfile=/var/run/hiawatha.pid
+healthcheck_timer=60
 
 depend() {
 	after firewall
 }
+
+stop() {
```

Comment:
supervise-daemon requires the default start,stop functions. This is just replicating in a fragile form what is done reliably with supervise-daemon.

Reply:
the current `initd` is broken.

Not only does it fail to start hiawatha - it does not stop it either due to the numerous processes hiawatha spawns.

Reply:
run `netstat -lptun` - hiawatha is not stopped

Reply:
![image](https://user-images.githubusercontent.com/30738253/64492263-91f8f100-d248-11e9-9420-a989f8b20f3c.png)

Reply:
when run as a reverse proxy it does not stop